projects/Mallard/src/components/article/types/article/webview.tsx

```diff
@@ -46,7 +46,7 @@ const WebviewWithArticle = ({
             {...webViewProps}
             originWhitelist={['*']}
             scrollEnabled={true}
-            source={{ html }}
+            source={{ html, baseUrl: '' }}
```

Comment:
Is it kind of a workaround for a bug/limitation? It looks like so, so I'd suggest adding a little comment to remind us later why this is here.